### DISCUSSION TITLE
[Bug] gemini apikey使用报错

### DISCUSSION THREAD

**Discussion by kanao7** [lobehub/lobehub#1808] [General Question] [answered]
### 💻 系统环境

Windows

### 📦 部署环境

Vercel / Zeabur / Sealos

### 🌐 浏览器

Edge

### 🐛 问题描述

同样的gemini apikey，在nextcaht和沉浸式翻译中填入可以正常使用，而在lobechat中却报错
{
  "error": {
    "message": "[GoogleGenerativeAI Error]: Error fetching from https://generativelanguage.googleapis.com/v1/models/gemini-pro:streamGenerateContent?alt=sse: [429 Too Many Requests] Quota exceeded for quota metric 'Generate Content API requests per minute' and limit 'GenerateContent request limit per minute for a region' of service 'generativelanguage.googleapis.com' for consumer 'project_number:736809909606'. [{\"@type\":\"type.googleapis.com/google.rpc.ErrorInfo\",\"reason\":\"RATE_LIMIT_EXCEEDED\",\"domain\":\"googleapis.com\",\"metadata\":{\"quota_limit_value\":\"0\",\"quota_metric\":\"generativelanguage.googleapis.com/generate_content_requests\",\"quota_limit\":\"GenerateContentRequestsPerMinutePerProjectPerRegion\",\"consumer\":\"projects/736809909606\",\"service\":\"generativelanguage.googleapis.com\",\"quota_location\":\"us-west4\"}},{\"@type\":\"type.googleapis.com/google.rpc.Help\",\"links\":[{\"description\":\"Request a higher quota limit.\",\"url\":\"https://cloud.google.com/docs/quota#requesting_higher_quota\"}]}]"
  },
  "provider": "google"
}

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
这个出错信息不是写的挺清楚了么：

[429 Too Many Requests] Quota exceeded for quota metric 'Generate Content API requests per minute' and limit 'GenerateContent request limit per minute for a region' of service 'generativelanguage.googleapis.com' for consumer 'project_number:736809909606'

**Reply by MangMax** [reply]
I waited two days to re-request the same problem, however I did not exceed the limit

**Comment by j0x3n** [comment]
> 这个出错信息不是写的挺清楚了么：
> 
> [429 Too Many Requests] Quota exceeded for quota metric 'Generate Content API requests per minute' and limit 'GenerateContent request limit per minute for a region' of service 'generativelanguage.googleapis.com' for consumer 'project_number:736809909606'

我也遇到了这个问题，同样的key在别的工具上例如沉浸式翻译就没问题，lobechat上就一直提示超限

**Comment by bestZwei** [comment]
我在gemini-pro-chat也遇到这个问题。

[a comment by an automated account is not shown]

**Comment by zasl** [comment]
刚注册的api也是这个提示
429 Too Many Requests

**Comment by zhou-jian-qq** [comment]
有可能是google检测到你这个apikey在多个地方使用， `此项目似乎通过将一个项目分为多个项目的方式来规避我们的配额限制。`

[a comment by an automated account is not shown]

**Comment by ludwikc** [comment]
I've generated brand new api key to check if that's the case, but I also get this error: 

`Error requesting Google service, please troubleshoot or retry based on the following information
`

```json

{
  "error": {
    "message": "[GoogleGenerativeAI Error]: Error fetching from https://generativelanguage.googleapis.com/v1beta/models/gemini-1.5-pro-latest:streamGenerateContent?alt=sse: [400 ] Invalid JSON payload received. Unknown name \"default\" at 'tools[0].function_declarations[0].parameters.properties[1].value': Cannot find field.\nInvalid JSON payload received. Unknown name \"default\" at 'tools[0].function_declarations[0].parameters.properties[3].value': Cannot find field.\nInvalid JSON payload received. Unknown name \"default\" at 'tools[0].function_declarations[0].parameters.properties[4].value': Cannot find field. [{\"@type\":\"type.googleapis.com/google.rpc.BadRequest\",\"fieldViolations\":[{\"field\":\"tools[0].function_declarations[0].parameters.properties[1].value\",\"description\":\"Invalid JSON payload received. Unknown name \\\"default\\\" at 'tools[0].function_declarations[0].parameters.properties[1].value': Cannot find field.\"},{\"field\":\"tools[0].function_declarations[0].parameters.properties[3].value\",\"description\":\"Invalid JSON payload received. Unknown name \\\"default\\\" at 'tools[0].function_declarations[0].parameters.properties[3].value': Cannot find field.\"},{\"field\":\"tools[0].function_declarations[0].parameters.properties[4].value\",\"description\":\"Invalid JSON payload received. Unknown name \\\"default\\\" at 'tools[0].function_declarations[0].parameters.properties[4].value': Cannot find field.\"}]}]"
  },
  "provider": "google"
}
```

![SIADLAK-Screenshot-202409031654hwlsp](https://github.com/user-attachments/assets/506689f7-6cd8-430c-8cba-8ce98a7d439b)
